CHANGES.rst

```diff
@@ -117,6 +117,15 @@ Read the `v2.0 migration guide <https://urllib3.readthedocs.io/en/latest/v2-migr
 * Fixed a reference cycle bug in ``urllib3.util.connection.create_connection()`` (`#2277 <https://github.com/urllib3/urllib3/issues/2277>`__).
 * Fixed a socket leak if ``HTTPConnection.connect()`` fails (`#2571 <https://github.com/urllib3/urllib3/pull/2571>`__).
 
+1.26.13 (2022-11-23)
+--------------------
+
+* Deprecated the ``HTTPResponse.getheaders()`` and ``HTTPResponse.getheader()`` methods.
```

Comment:
We already had a changelog entry for this, should we remove it? Also this entry does not mention that 2.1.0 will remove this.

Reply:
I think we should keep porting 1.26 changelogs as they are, so that anyone can track changes in stable versions from the main branch.

Having duplicates in 1.26.x and non-final 2.0 sections should be fine as long as we remember to remove them from the final 2.0.0 section when it is released.

Reply:
Yeah I was talking about removing the changelog entry in the 2.0 section.